pages/edit.js

```diff
@@ -0,0 +1,365 @@
+import Image from 'next/image';
+import styled from 'styled-components';
+import hook from '../public/hook.svg';
+import add from '../public/add-button.svg';
+import remove from '../public/delete-button.svg';
+import FormHeadline from '../components/EditFormComponents/FormHeadline';
+import EditBackgroundImage from '../components/EditFormComponents/EditBackgroundImage';
+import useLocalStorage from '../hooks/useLocalStorage';
+import { useRouter } from 'next/router';
+
+export function changeTimerChain(event, id, newTimerChain, setNewTimerChain) {
+  if (event.target.id === 'name') {
+    setNewTimerChain(
+      newTimerChain.map((newTimer) => {
+        if (newTimer.id === id) {
+          return {
+            ...newTimer,
+            name: event.target.value,
+          };
+        }
+        return newTimer;
+      })
+    );
+  } else if (event.target.id === 'seconds') {
+    setNewTimerChain(
+      newTimerChain.map((newTimer) => {
+        if (newTimer.id === id) {
+          return { ...newTimer, seconds: Number(event.target.value) };
+        }
+        return newTimer;
+      })
+    );
+  } else {
+    setNewTimerChain(
+      newTimerChain.map((newTimer) => {
+        if (newTimer.id === id) {
+          return { ...newTimer, minutes: Number(event.target.value) };
+        }
+        return newTimer;
+      })
+    );
+  }
+}
+
+export function handleAdd(newTimerChain, setNewTimerChain) {
+  const addId = newTimerChain.length + 1;
+  setNewTimerChain([
+    ...newTimerChain,
+    { id: addId, minutes: 5, seconds: 0, name: '' },
+  ]);
+}
+
+export function handleRemove(newTimerChain, setNewTimerChain) {
```

Comment:
Why is the function declared outside of the page? There would be no need to give it
```jsx
(newTimerChain, setNewTimerChain)
```
if it were inside of the edit page.